openlibrary/plugins/upstream/covers.py

```diff
@@ -84,7 +84,7 @@ def upload(self, key, i):
 
     def save(self, book, coverid, url=None):
         book.covers = [coverid] + [cover.id for cover in book.get_covers()]
-        book._save('{}/b/id/{}-S.jpg'.format(
+        book._save('{}/b/id/{}-M.jpg'.format(
```

Comment:
Was this change necessary?  It seems like this should be left as-is.

Reply:
Using M size with fixed height provides better quality images so one can zoom in and see the title.

Reply:
I wouldn't be surprised if this has some type of side effect.  Can we change it back for now?  Happy to merge this once that is done.